<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TARS" failures="2" tests="14" name="Test" time="3.582" errors="0" timestamp="2024-07-10T17:55:57 IST">
  <testcase classname="com.perrys.test.CreateMessage" name="sendMessage" time="2.856"/>
  <testcase classname="com.perrys.test.CreateMessage" name="sendReplyMessage" time="0.046"/>
  <testcase classname="com.perrys.test.CreateMessage" name="sendReplyMessageAgain" time="0.084"/>
  <testcase classname="com.perrys.test.CreateMessage" name="validateErrorForFromId" time="0.059"/>
  <testcase classname="com.perrys.test.CreateMessage" name="validateErrorForMessage" time="0.049"/>
  <testcase classname="com.perrys.test.CreateMessage" name="validateErrorForToId" time="0.051"/>
  <testcase classname="com.perrys.test.DeleteMessage" name="deleteUsingMessageId" time="0.035"/>
  <testcase classname="com.perrys.test.GetMessage" name="getUsingMessageId" time="0.056"/>
  <testcase classname="com.perrys.test.GetMessage" name="validateGetDeletedMessageId" time="0.019"/>
  <testcase classname="com.perrys.test.GetMessage" name="validateInvalidMessageId" time="0.02"/>
  <testcase classname="com.perrys.test.ListMessage" name="getListFromTo" time="0.129"/>
  <testcase classname="com.perrys.test.ListMessage" name="getListToFrom" time="0.049"/>
  <testcase classname="com.perrys.test.ListMessage" name="validateEitherFromOrToAsBlank" time="0.045">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;404&amp;gt; but was &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <404> but was <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at com.perrys.test.ListMessage.validateEitherFromOrToAsBlank(ListMessage.java:100)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- validateEitherFromOrToAsBlank -->
  <testcase classname="com.perrys.test.ListMessage" name="validateNoMessageBetweenFromTo" time="0.021">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;404&amp;gt; but was &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <404> but was <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at com.perrys.test.ListMessage.validateNoMessageBetweenFromTo(ListMessage.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- validateNoMessageBetweenFromTo -->
</testsuite> <!-- Test -->
